feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go

```diff
@@ -300,7 +300,7 @@ func (tc *testCase) configureDUT(t *testing.T) {
 		lacpPath.Replace(t, lacp)
 	}
 
-	tc.dut.Telemetry().Interface(tc.aggID).OperStatus().Await(t, 20*time.Second, opUp)
+	time.Sleep(5 * time.Second)
```

Comment:
same comment - we really want to wait for an expected state rather than blind sleep

Reply:
This will be removed later